cmd/src/main.go

```diff
@@ -61,16 +83,22 @@ type config struct {
 	AccessToken string `json:"accessToken"`
 }
 
-// readConfig reads the config file from the given path.
+// readConfig reads the config file from the given path, plus any in-scope environment variables
 func readConfig() (*config, error) {
 	cfgPath := *configPath
-	userSpecified := *configPath != ""
+	userSpecified := cfgPath != ""
 	if !userSpecified {
-		user, err := user.Current()
+		if cfgEnvPath := os.Getenv(ConfigEnvVar); cfgEnvPath != "" {
+			cfgPath = cfgEnvPath
+			userSpecified = true
+		}
+	}
+	if !userSpecified {
+		currentUser, err := user.Current()
 		if err != nil {
 			return nil, err
 		}
-		cfgPath = filepath.Join(user.HomeDir, "src-config.json")
+		cfgPath = filepath.Join(currentUser.HomeDir, ConfigFilename)
 	}
```

Comment:
I think this readConfig function has now become significantly more complex with the addition of the env getters logic.

I think adding utility functions which return first the CLI flag, second the env var would simplify this a ton and likely make the changes to `readConfig` here almost none.